src/services/Config/GlobalDefaultSettings.js

```diff
@@ -0,0 +1,70 @@
+/** Default values of the settings */
+let skipEnabled = true;
```

Comment:
the object below is meant to store the default values no? why do we need to add this here?

Reply:
this is meant to store the main value of the atomic setting, the leaf node

And then this variable is directly referenced inside our map. I put it here so it would be easy to change this and/or other settings in the future without finding where to change things deeply nested inside the map